### ISSUE TITLE
One Signal - Auo Prompt not working

### ISSUE THREAD

**Issue by applauzmedia** [myflashlab/ANE-LAB#145]
I'm using the FCM / One Signal ANE

settings.autoPrompt = false;  is not doing anything.  No matter what I set the settings to the prompt comes up right when the app opens. 

I have a custom dialog I have that opens the app opens. I want the user to Accept or Decline and trigger the prompt only if they accept.  

It seems that it automatically pops up because I have called   FCM.init();   

If I don't call FCM.init();   none of the One Signal code will work.

How do I get around this?

**Comment by myflashlab** [comment]
yes, as soon as ```FCM.init()``` is called the promotion window will be called. This is a very good thing to be fixed for next version. @nekofar

**Comment by applauzmedia** [comment]
Is there any workaround to this issue in the meantime ?

**Comment by myflashlab** [comment]
Hmm, of course you could try to initialize FCM later in your app when you have asked for the user consent.
